cEP-0021.md

```diff
@@ -0,0 +1,379 @@
+  `{section_name}{bear_name}.prof` so that if user want to make a sunburn
+  diagram of any bear of any section they can easily do that with `sankeviz`
+  module or any subtitute module.
+
+- If `No` the then it will check the first value of a list return by
+  `setup_profiler` is "True" or the "filename or file-path" i.e.
+  `--profile-bears "True"` or \`--profile-bears "<filename or file-path>"``
+
+**If `True`:**
+
+1. Then it further calls a `setup_profiler_table()` method which first
+   create a `temp` file (`fd` here) and calls a `pstats_config()` method
+
+```python
+def setup_profiler_table(self, prof,
+                         profile_bears, check_trim: bool = False):
+    fd, path = tempfile.mkstemp()
+    try:
+        with fdopen(fd, 'r+') as stream:
+            ps = self.pstats_config(prof, stream, profile_bears)
+            # print(vars(ps))
+            ps.print_stats()
+            stream.flush()
+            stream.seek(0)
+            lines = stream.readlines()
+            req_lines = []
+            table_header = []
+            for line in lines:
+                if ('function calls' not in line and
+                    'listing order was used' not in line and
+                    'Ordered by' not in line and
+                        line.strip(' \n\t') != ''):
+                    req_lines.append(line)
+                elif line.strip(' \n\t') != '':
+                    table_header.append(line.strip())
+            channel_values = [x for x in [y.split(' ') for y in
+                                          req_lines] if x and not x == '\n']
+            table_data = []
+            final_table_data = []
+            for x in channel_values:
+                table_data.append(list(filter(lambda a: a != '', x)))
+            for x in table_data:
+                start = ''
+                flag = 0
+                row = []
+                for y in x:
+                    if y.startswith('{'):
+                        flag = 1
+                    if flag == 1:
+                        start = start + y + ' '
+                    elif y != '':
+                        t = ' '.join(y.split())
+                        row.append(t.strip())
+                if start != '':
+                    row.append(start.strip())
+                final_table_data.append(row)
+            colored_table_data = []
+            colors = ['red', 'white', 'blue', 'yellow', 'magenta', 'green']
+            final_table_data = final_table_data if check_trim else (
+                final_table_data[:15])
+            for x in final_table_data:
+                row = []
+                for index, y in enumerate(x):
+                    row.append(colored(y, colors[index]))
+                colored_table_data.append(row)
+            table = AsciiTable(colored_table_data)
+            for line in table_header:
+                print(line)
+            cprint(table.table)
+    finally:
+        remove(path)
+```
```

Comment:
I don't think you need all this, just mention that it will dump the profiler results nicely using whatever classes on the CLI